README.md

```
@@ -1,61 +1,409 @@
--   [Skyra Invite Link](https://invite.skyra.pw)
--   [Support Server](https://join.skyra.pw)
--   [Patreon](https://donate.skyra.pw/patreon)
+</div>
 
-## Buy us some doughnuts
+<details>
+<summary><kbd>Table of contents</kbd></summary>
 
-Skyra Project is open source and always will be, even if we don't get donations. That said, we know there are amazing people who
-may still want to donate just to show their appreciation. Thanks you very much in advance!
+#### TOC
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)